op/coreaudio.c

```diff
@@ -267,10 +65,14 @@ static OSStatus coreaudio_play_callback(void *user_data,
 					AudioBufferList *buflist)
 {
 	int count = nframes * coreaudio_format_description.mBytesPerFrame;
```

Comment:
Would this always be equal to `buflist->mBuffers[0].mDataByteSize`? Maybe we can use that directly when `memcpy()`?